vimeo-networking/build.gradle

```diff
@@ -14,10 +17,12 @@ buildscript {
     }
     dependencies {
         classpath 'net.ltgt.gradle:gradle-apt-plugin:0.6'
+        classpath 'gradle.plugin.de.fuerstenau:BuildConfigPlugin:1.1.8'
```

Comment:
Is everyone fine with adding this plugin? it appears to be the only build config plugin for java projects https://github.com/mfuerstenau/gradle-buildconfig-plugin